report/sections/directembedding.tex

```diff
@@ -100,13 +100,14 @@ \subsubsection{@passThrough} % (fold)
 \label{sub:passThrough}
 The \texttt{@passThrough} annotation is useful when certain methods should not be reified in the ReificationTransformation.
 By default, any invoked method in a DSL program should be reified during ReificationTransformation.
-If a method is missing a reification annotation, the ReificationTransformation will return with a compilation error.\
+If a method is missing a reification annotation, the \texttt{ReificationTransformation} will return with a compilation error.\
 A DSL author can annotate a method with \texttt{@passThrough} if she wants the functionality of the method to be preserved from the shallow DSL.\
 Slick-direct uses this annotation in a few cases.
 
 % subsection passThrough (end)
 \subsubsection{customLifts} % (fold)
 \label{sub:customLifts}
+% Can't we solve this with implicit conversions? E.g., lift[T, U](t: T)(c: Converter[T, U]): U
```

Comment:
After some thinking. Yes, I believe we could do that, although I am not entirely sold that it's a good idea.

I agree that the fewer configuration parameters are needed in the library, the better. However, the `Converter` might turn out to be trickier to work with for the DSL author than `customLifts`.

@vjovanov